requirements.txt

```diff
@@ -40,7 +40,7 @@ qrcode==6.1
 reportlab==3.5.59 # version < 3.5.54 are not compatible with Pillow 8.1.2 and 3.5.59 is bullseye
 requests==2.22.0
 vobject==0.9.6.1
-Werkzeug==0.16.1
+Werkzeug==2.0.2
```

Comment:
This will most likely be a no for this change: requirements versions are frozen, and defined (most of the time) by the minimal version of the python package in the stable version of debian and ubuntu when the odoo version is released. 

In ubuntu 20.4, the package version is 0.16.1: https://ubuntu.pkgs.org/20.04/ubuntu-main-amd64/python3-werkzeug_0.16.1+dfsg1-2_all.deb.html

@d-fence may confirm that point

And anyway the runbot is red

Reply:
aah, the "joy" of debian ...

Reply:
I confirm the point.